src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -54,7 +54,6 @@ const useSimpleWebsocket = (url, protocol = null) => {
     };
 
     socket.onmessage = (event: MessageEvent) => {
-      console.log('onmessage', event);
```

Comment:
Unnecessary logs (debug purpose)

Reply:
mas @arilukmanp this is deletion mas

Reply:
mas @arilukmanp this is deletion mas, so on current code this should not be there anymore

Reply:
Sorry, misreading, lack of focus